### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by flora-hofmann-frequenz**
Quick info on how to use the new tooling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,51 @@ The following platforms are officially supported (tested):
...
+
+```python
+energy_reading = await cumulative_energy(client=client,
+                                     microgrid_id=1,
+                                     component_id=100,
+                                     start_time=datetime.fromisoformat("2024-09-01T00:00:00"),
+                                     end_time=datetime.fromisoformat("2024-09-30T00:00:00"),
+                                     use_active_power=True,
+                                     resolution=10,
+    )
```
Comment (cwasicki) [inline README.md:68]: The format of the above looks off.

**Review by cwasicki — DISMISSED** [review]
LGTM, just formatting looks a bit weird.

File: README.md (hunk 1 above)
Comment (flora-hofmann-frequenz) [reply README.md:68]: I moved it in a bit more then one also does not have to scroll too far to the right.

Comment (cwasicki) [reply README.md:68]: ok, I guess black would give something different but doesn't matter too much.

**Review by cwasicki — APPROVED** [review]

**Comment by flora-hofmann-frequenz** [comment]
`black` does not run on the readme as far as I know.

**Pull request merged**